units/XNA0303/XNA0303_unit.bp

```diff
@@ -302,7 +303,7 @@ UnitBlueprint {
                 Air = 'Air|Land|Water',
                 Land = 'Air|Land|Water',
             },
-            FiringTolerance = 0,
```

Comment:
firingTolerance is identical to UEF ASF, is there a reason why this is better? It can stay at 2 degrees i suppose but this doesnt seem like a change that would do much as it already uses the aimboneTransform.